ci-operator/step-registry/ipi/conf/nutanix/preload-image/ipi-conf-nutanix-preload-image-commands.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f ${CLUSTER_PROFILE_DIR}/secrets.sh ]]; then
+  NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+else
+  NUTANIX_AUTH_PATH=/var/run/vault/nutanix/secrets.sh
+fi
+declare prism_central_host
+declare prism_central_port
+declare prism_central_username
+declare prism_central_password
+# shellcheck source=/dev/null
+source "${NUTANIX_AUTH_PATH}"
+
+echo "$(date -u --rfc-3339=seconds) - Import Image"
+pc_url="https://${prism_central_host}:${prism_central_port}"
+un="${prism_central_username}"
+pw="${prism_central_password}"
+
+coreos_location=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.nutanix.formats.qcow2.disk.location')
+coreos_release=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.nutanix.release')
+preload_image_name=qe-preload-$coreos_release.qcow2
+
+data="{
+  \"kind\": \"image\"
+}"
+api_ep="${pc_url}/api/nutanix/v3/images/list"
+image_json=$(curl -ks -u "${un}":"${pw}" -X POST "${api_ep}" -H "Content-Type: application/json" -d @-<<<"${data}")
+image_uuid=$(echo "${image_json}" | jq -r ".entities[] | select (.spec.name == \"${preload_image_name}\") | .metadata.uuid")
+
+if [[ -n "${image_uuid}" ]]; then
+  echo "$(date -u --rfc-3339=seconds) - preload image alread exist"
+  # preserve image uuid to delete
+  echo "$image_uuid" > "${SHARED_DIR}/preload-image-delete.txt"
```

Comment:
If the image already exists maybe other jobs is running, I do not think we should delete it in this job.

Reply:
Got it, I added an uniq name to avoid conflict, pls have a look again, thanks.